README.md

```diff
@@ -120,6 +130,8 @@ See [https://github.com/kelseyhightower/kubernetes-the-hard-way](https://github.
 
 ## Books
 
+CapGemini has a deal with SkillSoft - lots of good quality eBook content is available online and offline if you have ann iOS device - no android support yet. More details [here] (http://talent.capgemini.com/uk/pages/HR/career_development/learning_development/review_the_capgemini_learning_environment/skillsoft/)
```

Comment:
this is an open repo - i'd like to keep most of the stuff public if we can, or open it up if its not. However dont think we can with this one - so can we remove this bit?